test/test_dosemu.py

```diff
@@ -4802,7 +4802,7 @@ def test_freecom_build(self):
         # Note:
         #     We have to avoid runDosemu() as this test is non-interactive
         args = ["-q", "-K", ".", "-E", "build.bat"]
-        results = self.runDosemuCmdline(args, cwd=root, timeout=450)
+        results = self.runDosemuCmdline(args, cwd=root, timeout=2000)
```

Comment:
Wow, not that fast...
But passes.

Reply:
I'm a bit puzzled here since it passed before in a faster time. Then I ran the test locally and it took ~900 seconds, also for the commit from last week. Pure CPUSIM takes about 23 mins locally (~1350 seconds), JIT 195 sec.

So the 2000 was conservative but in the end 600 would have worked too (github servers being faster than my laptop). I'm rerunning the test from last week on CI to see if that still passes.

Reply:
indeed, https://github.com/bartoldeman/dosemu2/commit/4f390c86ea23d6ae9e5d138ebc54849a490b9b34 failed today with a timeout but passed last week. Must have been a fast CI server.

Reply:
Thanks, this gives a good
overview for expectations.
So its still much closer to
sim than to jit, meaning
that interp is not the biggest
part of the deal.
I actually was hopping that
it is. :(

Reply:
it's at least consistent with the 30% I measured in the beginning.

The good news is that a bunch of optimizations in `codegen-sim.c` are now easier to do, there is quite a bit of improvement possible there, to cut down on the overhead of processing every intermediate op, and some unfinished lazy flag handling.